Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -129,5 +129,17 @@ private void UpdateStabilizationPlane()
                 HolographicSettings.SetFocusPointForFrame(newFocusPointPosition, -gazeDirection);
             }
         }
+
+        /// <summary>
+        /// Sends message to the object currently focused on.
+        /// </summary>
+        /// <param name="message">Message to send</param>
+        public void SendMessageToFocusedObject(string message)
```

Comment:
If you make the focusedObject public:
1. Any other class can also use it for hands / gestures or voice. They can do whatever they choose to do with it. 
2. They also don't need to depend on SendMessage VS SendMessageUpwards or other variants.
3. They could use it a more polymorphic way instead of this message way.